GitUI/CommandsDialogs/AboutBoxDialog/FormContributors.cs

```diff
@@ -26,7 +26,7 @@ private TextBox GetNewTextBox()
             TextBox tb = new TextBox();
             tb.BackColor = Color.White;
             tb.Dock = DockStyle.Fill;
-            tb.Font = new Font("Segoe UI", 9.75F, FontStyle.Regular, GraphicsUnit.Point, ((byte)(0)));
+            tb.Font = new Font("Segoe UI", 9.75F, FontStyle.Regular, GraphicsUnit.Point,  (byte)0);
```

Comment:
:bulb: This could be filed as a StyleCop Analyzers bug. It introduced a second space after `,`.

Reply:
Shouldn't the second space be captured by some other rule?

Reply:
I would have thought so, but looking through the rules we haven't enabled I don't see anything.

Reply:
It's caught by SA1025. The reason I call this a :bug: is the original code only had one space, so the fix is not only introducing a violation of SA1025 but it's also changing whitespace unrelated to the rule getting corrected.

Reply:
In that case perhaps there are two bugs. One that introduced the additional space, and another in SA1025 that didn't pick up the multiple spaces in this case. SA1025 is enabled on this code and it flags multiple spaces elsewhere in the file, but not between a comma and the following argument.